docs/verifier/query-builder.md

```diff
@@ -2,59 +2,209 @@
+
+2. **Selective disclosure:** Use this when you need the prover to reveal specific data stored in their credential. This option discloses the actual value of an attribute to the verifier. For instance, instead of just requesting proof proving legal age, it would disclose the prover's actual age to the verifier.
+
+Choose "Condition" for privacy-preserving verification, or "Selective disclosure" for cases where revealing specific information is necessary.
+
+### Operator
+
+In conditional queries, the "Operator" input defines the logical operation to be applied. The supported operators are:
+
+- **Equal:** Checks if the value is equal to the specified value.
+- **Not Equal:** Verifies if the value is not equal to the specified value.
+- **Less Than:** Assesses that the value is less than the given value.
+- **Greater Than:** Evaluates if the value is greater than the specified value.
+- **Matches one of the values:** Determines if the value is within a given set of values.
+- **Matches none of the values:** Checks if the value is not within a specified set.
 
-Then, one can copy and paste the snippet on the verifier API `GetAuthRequest()` function.
+Choose the appropriate operator based on the condition you want to validate.
 
-:::info
+### Attribute value
 
-The idea behind the Query Builder is to facilitate the query elaboration process by generating the whole section relative to "Request for specific proof" of the auth request endpoint code, as it can be seen on the [<ins>Run a Verifier tutorial</ins>](/docs/verifier/verification-library/verifier-set-up.md#verifier-server-setup), more specifically on point 3.
+The "Attribute value" field allows verifiers to input the specific value or range of values used to compare with the selected attribute in the prover's credential. This field adapts to the type of the chosen attribute, accommodating various data types such as strings, numbers, booleans, dates, and more. This ensures that the value provided is compatible with the selected operator and the nature of the attribute.
+
+### Issuer DID
+
+This field in the query form is used to specify the Decentralized Identifier (DID) of the issuer that is trusted by the verifier. By entering a specific DID, the verifier can confirm that the credential presented by the prover was indeed issued by the specified issuer.
+
+:::caution
+
+Using `*` as a wildcard to accept credentials from any issuer is possible but not recommended, particularly in production environments. This approach may compromise the query's security and integrity, as it will validate credentials issued by any issuer.
+
+:::
+
+:::caution
+
+Currently, the system supports specifying only a single DID. This means users have the choice to either provide one specific issuer DID or use `*` to allow any issuer. However, the former is strongly advised for maintaining robust verification standards.
```

Comment:
not the system, but the query builder. On protocol level you can have more then one specified issuer

Reply:
I thought this was a limitation in the protocol. I don't understand why we have this limited in the Query Builder then...

c.c @elias-garcia @javip97

Reply:
Done

Reply:
We limited it in the query builder due to time constraints, but I'm aware that the protocol supports more than one issuer. I've created a ticket for this in Jira.